float32 x_rect
float32 y_rect
float32 width
float32 height
---
